expo/gbe/views/review_volunteer_view.py

```diff
@@ -1,103 +1,44 @@
 from django.contrib.auth.decorators import login_required
-from django.shortcuts import (
-    get_object_or_404,
-    render,
-)
-from django.http import HttpResponseRedirect
-from django.core.urlresolvers import reverse
-from expo.gbe_logging import log_func
-from gbe.functions import (
-    validate_perms,
-    get_conf,
-)
-from gbe.models import (
-    BidEvaluation,
-    Conference,
-    Volunteer,
-)
+from gbe.models import Volunteer
+
 from gbe.forms import (
     BidEvaluationForm,
-    BidStateChangeForm,
+    VolunteerBidStateChangeForm,
```

Comment:
Okay, restoring that. I was wondering why this wasn't using that form. However, I see that the action form for this calls volunteer_changestate_view, which is going to try to build a VolunteerBidStateChangeForm off of the data we send it. This seems kind of weird to me. Is this intentional?